### DISCUSSION TITLE
Falcon api.reposify.com/v2 upcoming deprecation

### DISCUSSION THREAD

**Discussion by TomerSeemplicity** [CrowdStrike/falconpy#1236] [Q&A] [answered]
Hey Guys,

I’ve received some information from our peers regarding the upcoming deprecation of the api.reposify.com/v2 endpoints. Currently, I'm using these endpoints to retrieve footprints and services for each footprint.

From what I’ve been informed, the new recommended approach is to use https://api.crowdstrike.com/fem/queries/external-assets/v1 instead of https://api.reposify.com/v2/company/my-comp.com/footprints.

Could you please confirm if this API is indeed being deprecated? If so, what is the recommended best practice for pulling vulnerabilities for external assets? So far, I haven't found a suitable method for doing this.

Thanks!

**Comment by TomerSeemplicity** [comment]
@jshcodes , is there any chance you've heard anything about that?

**Reply by jshcodes** [reply, marked as answer]
Hi @TomerSeemplicity -

I have confirmed the legacy endpoint is deprecated starting this quarter.  

The new endpoint [query_external_assets](https://www.falconpy.io/Service-Collections/Exposure-Management.html#query_external_assets) is available in FalconPy as part of the [Exposure Management](https://www.falconpy.io/Service-Collections/Exposure-Management.html) service collection. (Filters can be found [here](https://www.falconpy.io/Service-Collections/Exposure-Management.html#available-filters-exact-match), there are several lists depending on the matching the filter supports.)

Thank you for the question, let us know if you have any issues! 😄
